floodlight/core/pitch.py

```diff
@@ -147,3 +153,967 @@ def center(self):
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+
+        # goals
+        # left goal
+        ax.plot(
+            [xmin - goal_height, xmin - goal_height],
+            [y_value_goal_lower_post, y_value_goal_upper_post],
+            color=goal,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        ax.plot(
+            [xmin - goal_height, xmin],
+            [y_value_goal_lower_post, y_value_goal_lower_post],
+            color=goal,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        ax.plot(
+            [xmin - goal_height, xmin],
+            [y_value_goal_upper_post, y_value_goal_upper_post],
+            color=goal,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        # right goal
+        ax.plot(
+            [xmax + goal_height, xmax + goal_height],
+            [y_value_goal_lower_post, y_value_goal_upper_post],
+            color=goal,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        ax.plot(
+            [xmax + goal_height, xmax],
+            [y_value_goal_lower_post, y_value_goal_lower_post],
+            color=goal,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        ax.plot(
+            [xmax + goal_height, xmax],
+            [y_value_goal_upper_post, y_value_goal_upper_post],
+            color=goal,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        # baseline
+        ax.plot(
+            [xmax, xmax],
+            [ymin, ymax],
+            color=innerline,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+        # baseline
+        ax.plot(
+            [xmin, xmin],
+            [ymin, ymax],
+            color=innerline,
+            linewidth=linewidth,
+            scalex=False,
+            scaley=False,
+            zorder=0,
+        )
+
+        # remove labels and ticks
+        ax.xaxis.set_major_locator(matplotlib.ticker.NullLocator())
+        ax.yaxis.set_major_locator(matplotlib.ticker.NullLocator())
+        ax.set_xticklabels([])
+        ax.set_yticklabels([])
+
+        if save:
+            path = os.path.abspath(
+                os.path.join(
+                    os.path.abspath(
+                        os.path.join(
+                            os.path.dirname(os.path.abspath(__file__)), os.pardir
+                        )
+                    ),
+                    os.pardir,
+                )
+            )
+            plt.savefig(str(path) + "_handball_pitch.png")
```

Comment:
Implemented.

Reply:
<img width="710" alt="image" src="https://user-images.githubusercontent.com/73299378/156393009-e44e562f-0247-4a72-97c9-86d74cd189b6.png">
As you probably know the savefig() method accepts some more arguments. Do we want to provide the possibility to pass these arguments as well? If we also pass the kwargs to the savefig() method we need to filter them since savefig() accepts different arguments than for instance plt.plot().